examples/generate_bom/compute_cogs.py

```diff
@@ -115,7 +115,11 @@ def fetch_price_for_part(part_number: str) -> dict[int, float]:
     )
     parser.add_argument(
         "--output-file",
-        help="The path to the output file. Defaults to stdout, i.e. printing to the console.",
+        help="The relative path to the output file. Defaults to stdout, i.e. printing to the console.",
+    )
+    parser.add_argument(
+        "--workspace",
+        help="The workspace directory to put the output file in.",
     )
```

Comment:
I would prefer a default of `.` here, so that this doesn't crash later at :196 if not supplied.